src/lib/viewers/text/TextBaseViewer.js

```diff
@@ -55,17 +62,23 @@ class TextBaseViewer extends BaseViewer {
         let newFontSize = 0;
 
         if (inOrOut === 'in') {
-            newFontSize = `${size + 10}%`;
+            newFontSize = size + 10;
         } else if (inOrOut === 'out') {
-            newFontSize = `${size - 10}%`;
+            newFontSize = size - 10;
         }
 
-        el.style.fontSize = newFontSize;
+        el.style.fontSize = `${newFontSize}%`;
         this.emit('zoom', {
             zoom: newFontSize,
             canZoomIn: true,
             canZoomOut: true,
         });
+        this.zoomControls.setCurrentScale(newFontSize / 100);
+    }
+
+    getFontSize() {
+        const el = this.containerEl.querySelector('.bp-text');
+        return parseInt(el.style.fontSize, 10) || 100;
```

Comment:
Should we be using a class property rather than the DOM to track the current state?

Reply:
I agree that the font size/scale could be better represented than the DOM computedStyles. I'll address that in an follow up PR. This was just a convenience method based on what was already happening in L53-43, which I forgot to replace with this 😛